docs/api-reference/next.config.js/custom-page-extensions.md

```diff
@@ -14,6 +14,11 @@ module.exports = {
 }
 ```
 
+> Configuring `pageExtensions` also affects `_document.js`, `_app.js` as well as files under `pages/api/`.
+>
```

Comment:
This has to be one large block otherwise the site will render in a weird way (multiple blocks)